languages/go/internal/cinterface/bitwarden_library_custom.go

```diff
@@ -0,0 +1,58 @@
+//go:build custom
+// +build custom
+
+package cinterface
+
+import (
+	"fmt"
+	"unsafe"
+)
+
+/*
+#cgo LDFLAGS: -lbitwarden_c
+#cgo linux LDFLAGS: -L/usr/local/lib -L/usr/lib
+#cgo darwin LDFLAGS: -L/usr/local/lib -L/usr/lib
```

Comment:
This seems to be the only difference for custom? Is there a reason to not always attempt to find the binary in a local`./lib` directory?

Reply:
Personally I've always used the `./lib` directory to copy the `libbitwarden_c.dylib` file and then ran `go build`, so I've never used either the `bitwarden_library_custom.go` or the `build.sh` script, maybe they are not needed anymore?

Reply:
I agrree, let's remove it